src/components/ConfirmationBox/container.js

```diff
@@ -0,0 +1,23 @@
+import ConfirmationBox from './ConfirmationBox';
+import {setActualItem} from './actions';
+import {connect} from 'react-redux';
+
+const mapStateToProps = state => {
```

Comment:
if all you are doing is returning you can simplify this

```
const mapStateToProps = state => ({
  prediction: state. prediction,
  storeList: state.storeList
});
```

similarly with dispatch.